charts/admission-controller/README.tpl

```diff
@@ -12,365 +12,92 @@ $ pre-commit run -a
 
+## Overview
 
+[{{ .Project.Name }}]({{ .Project.URL }}) provides Audit Logging and optional Image Scanning capabilities to secure your Kubernetes environment.
 
-### Prerequisites
-{{ range .Prerequisites }}
-- {{ . }}
-{{- end }}
-
-
-
-###  Installing the Chart
-
-To install the chart with the release name `{{ .Release.Name }}`:
+Use the [sysdig-deploy](../sysdig-deploy/README.md) parent chart to deploy the Admission Controller and any other subcomponents. Do not deploy subcharts directly.
 
-```console
-$ helm upgrade --install sysdig-{{ .Release.Name }} {{ .Repository.Name }}/{{ .Chart.Name }} -n {{ .Release.Namespace }}{{ with .Chart.Version }} --version={{.}}{{ end }}
-```
-
-The command deploys {{ .Project.App }} on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
-
-> **Tip**: List all releases using `helm list`
-
-
-
-### Uninstalling the Chart
-
-To uninstall/delete the `{{ .Release.Name }}`:
+To deploy the Admission Controller, follow the installation instructions given in  [Install Kubernetes Audit Logging](https://docs.sysdig.com/en/docs/installation/sysdig-secure/install-agent-components/kubernetes/install-kubernetes-audit-logging/).
 
-```console
-$ helm uninstall sysdig-{{ .Release.Name }} -n {{ .Release.Namespace }}
-```
+## Use Cases
 
-The command removes all the Kubernetes components associated with the chart and deletes the release.
+### Kubernetes Audit Logging
 
-{{ if .Chart.Values -}}
+This chart is primarily responsible for enabling Kubernetes audit logging so that Sysdig Secure can audit the following:
 
-## Configuration
+- Creation and destruction of pods, services, deployments, and DaemonSets.
+- Creating, updating, and removing ConfigMaps or secrets.
+- Attempts at subscribing to the changes to any endpoint.
 
-The following table lists the configurable parameters of the `{{ .Chart.Name }}` chart and their default values.
+For deployment instructions, including common deployment configurations related to proxies and certificates, see [Install Kubernetes Audit Logging](https://docs.sysdig.com/en/docs/installation/sysdig-secure/install-agent-components/kubernetes/install-kubernetes-audit-logging/).
 
-{{ .Chart.Values }}
+### (Legacy Option) Image Scanning Using Scanning Engine V1
 
-Specify each parameter using the **`--set key=value[,key=value]`** argument to `helm upgrade --install`. For example:
+If you use the [Legacy Scanning Engine](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/) instead of the new Vulnerability Management engine in Sysdig Secure, you can deploy the `admission-controller` chart with old scanning options enabled and use [admission controller policies](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/admission-controller/) to reject container images that do not fulfill the policy requirements from the cluster before being scheduled.
```

Comment:
We could improve the links now that we have a page in the docs for the Legacy Scanning AC, including install instructions:

```suggestion
If you use the [Legacy Scanning Engine](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/) instead of the new Vulnerability Management engine in Sysdig Secure, you can [deploy the `admission-controller` chart](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/admission-controller/#installation) with old scanning options enabled and use [admission controller policies](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/admission-controller/#create-admission-controller-policies) to reject container images that do not fulfill the policy requirements from the cluster before being scheduled.
```